.github/workflows/unitTestsWindows.yml

```diff
@@ -24,7 +24,8 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node_version }}
-          cache: npm
+          cache: 'npm'
+          cache-dependency-path: '**/package-lock.json'
```

Comment:
I was seeing a lot of weirdness in the GHA caching of windows.  This seems to make it happy and is the suggested config from the setup-node repo